djpsa/halo/management/commands/callbacks_deleted.py

```diff
@@ -0,0 +1,7 @@
+from djpsa.sync.management.commands.base_callbacks_deleted import \
```

Comment:
Let's not use this pattern for these 3 commands. It will make it a pain to maintain and to add new PSAs. We use this pattern a lot in views in django-topleft (a child view for the PSA inherits from the generic view) and it's a lot more complicated than the strategy pattern.

Please replace this with 3 commands in the sync app, which loads the appropriate PSA strategy somehow (we can discuss the possibilities for the "somehow") and calls the necessary commands on the strategy.

Reply:
Will do

Reply:
Ok, I've reintroduced the provider to handle this so djpsa can get information it needs depending on the app mode its running in.